lib/utils/string_extension.dart

```diff
@@ -47,4 +47,30 @@ extension StringCasingExtension on String {
   String toTomMatrixId() {
     return '@$this:tom-dev.xyz';
   }
+
+  List<String> getMentionsFromMessage() {
+    final RegExp regex = RegExp(r"@\[([^\]]+)\]");
+    final Iterable<Match> matches = regex.allMatches(this);
+
+    final List<String> mentions = [];
+    for (final Match match in matches) {
```

Comment:
IMO, you can make code easier to read by using `matches.where((match) => match.group(0) != null)`

Reply:
I don't want to have duplicate mentions